node-template/src/service.rs

```diff
@@ -111,22 +103,21 @@ pub fn new_full<C: Send + Default + 'static>(config: Configuration<C, GenesisCon
 		let select_chain = service.select_chain()
 			.ok_or(ServiceError::SelectChainRequired)?;
 
-		let babe_config = babe::BabeParams {
-			keystore: service.keystore(),
-			client,
+		let aura = aura::start_aura::<_, _, _, _, _, AuraPair, _, _, _>(
+			aura::SlotDuration::get_or_compute(&*client)?,
+			client.clone(),
```

Comment:
We should pass `GrandpaBlockImport` here instead of the client. We need to store the `grandpa_block_import` in `import_setup` as we do for `grandpa_link`.

Reply:
done